ui/src/app/components/sidenav-rule/sidenav-rule.component.ts

```diff
@@ -22,8 +23,20 @@ export class SidenavRuleComponent implements OnInit {
   ruletype: string = ''
   viewRuleData: any = []
   viewRuleFlag: boolean = false
+  shardedMigration: boolean = false
 
   ngOnInit(): void {
+    this.data.conv.subscribe({
+      next: (res: IConv) => {
+        console.log(1, res)
+        const keys = Object.keys(res.SpSchema) as Array<string>
+        if (res.SpSchema[keys[0]].ShardIdColumn != '') {
```

Comment:
is this the only way to find if a migration is sharded ? There should be a flag which explicitly calls it out .

Reply:
Added IsSharded flag